src/operator/tensor/indexing_op.cc

```diff
@@ -13,10 +13,46 @@ DMLC_REGISTER_PARAMETER(TakeParam);
 DMLC_REGISTER_PARAMETER(OneHotParam);
 
 NNVM_REGISTER_OP(Embedding)
-.MXNET_DESCRIBE("Map integer index to vector representations (embeddings)."
-                " Those embeddings are learnable parameters. For a input of shape"
-                " (d1, ..., dK), the output shape is (d1, ..., dK, output_dim)."
-                " All the input values should be integers in the range [0, input_dim).")
+.describe(R"code(Maps integer indices to vector representations (embeddings).
+
+This operator maps words to real-valued vectors in a high-dimensional space,
+these are called as word embeddings.
+These word embeddings can learn semantic and syntactic information of the words
+i.e, similar words are close to each other in this space and dissimilar words far apart.
```

Comment:
This is an overstatement. It's sometimes, but not always true. I would state it more softly. 

Also i.e. means "that is". You mean "for example" which is ".e.g.". Also don't start a sentence with Latin shorthand. Just use english. 

Better version: 

"For example, it's been noted that in the learned embedding spaces, similar words tend to be close to each other and dissimilar words far apart."